src/styles/style.css

```diff
@@ -0,0 +1,126 @@
+:root {
+  --main-color: #3d4e61;
+  --border-color: #e1e7ed;
+}
+
+/* #region font */
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: bold;
```

Comment:
The `font-style` should be set to 'normal' instead of 'bold' for the bold font to avoid confusion. The `font-weight` property already specifies the weight as 700.